src/main/scala/net/ruippeixotog/scalascraper/browser/Browser.scala

```diff
@@ -15,6 +15,16 @@ trait Browser {
   def parseFile(path: String, charset: String): Document = parseFile(new File(path), charset)
   def parseFile(path: String): Document = parseFile(new File(path), "UTF-8")
 
+  def parseResource(name: String, charset: String = "UTF-8"): Document = {
+    val stream = getClass.getResourceAsStream(name)
+    val document = parseInputStream(stream, charset)
+    stream.close()
+    document
+  }
+
+  /** Does not close the InputStream after reading */
+  def parseInputStream(inputStream: InputStream, charset: String = "UTF-8"): Document
```

Comment:
Just curious, why did you chose not to close the `InputStream` after reading? Do you have a specific use case in mind? If I'm not mistaken, both `Browser` implementations will always consume the entire stream, right?